### DISCUSSION TITLE
overriding the default lsp keymaps

### DISCUSSION THREAD

**Discussion by jrnxf** [LazyVim/LazyVim#106] [Q&A] [answered]
I'm contemplating switching over from telescope to [fzf-lua](https://github.com/ibhagwan/fzf-lua), but I'm not sure the right way to override the telescope keymaps set [here](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/lsp/keymaps.lua).

@folke any recommendation here?

Also, as a side note, I might have time soon to add fzf-lua as a provider in trouble, so that I can send my results from fzf-lua to trouble like you can with telescope. Would you be interested in me PRing that?

**Comment by folke** [comment, marked as answer]
You can check the docs on that here: https://lazyvim.github.io/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps

feel free to create a PR for fzf-lua!
